src/test/java/dk/kb/present/solr/EmbeddedSolrTest.java

```diff
@@ -0,0 +1,592 @@
+		assertEquals("1831",record.getFieldValue("production_date_end"));
+		assertEquals(6691996, record.getFieldValue("file_size"));
+		assertEquals(1812, record.getFieldValue("image_height"));
+		assertEquals(1227, record.getFieldValue("image_width"));
+
+		//multivalue field
+		// creator_name
+		Collection<Object> creatorName = record.getFieldValues("creator_name");
+		assertEquals(1, creatorName.size());
+		assertTrue(creatorName.contains("Clemens, Johann Friderich"));
+
+		// creator_full_name
+		Collection<Object> creatorFullName = record.getFieldValues("creator_full_name");
+		assertEquals(1, creatorFullName.size());
+		assertTrue(creatorFullName.contains("Johann Friderich Clemens"));
+
+		// creator_family_name
+		Collection<Object> creatorFamilyName = record.getFieldValues("creator_family_name");
+		assertEquals(1, creatorFamilyName.size());
+		assertTrue(creatorFamilyName.contains("Clemens"));
+
+		// creator_given_name
+		Collection<Object> creatorGivenName = record.getFieldValues("creator_given_name");
+		assertEquals(1, creatorGivenName.size());
+		assertTrue(creatorGivenName.contains("Johann Friderich"));
+
+		// creator_terms_of_address
+		Collection<Object> creatorTermsOfAddress = record.getFieldValues("creator_terms_of_address");
+		assertEquals(1, creatorTermsOfAddress.size());
+		assertTrue(creatorTermsOfAddress.contains("kobberstikker"));
+
+		// type_of_resource
+		Collection<Object> typeResources = record.getFieldValues("type_of_resource");
+		assertEquals(2,typeResources.size());
+		assertTrue(typeResources.contains("Billede, Todimensionalt billedmateriale"));
+		assertTrue(typeResources.contains("Grafik"));
+
+		// topic
+		Collection<Object> topic = record.getFieldValues("topic");
+		assertEquals(2,topic.size());
+		assertTrue(topic.contains("Poul Egede. 1911,7507."));
+		assertTrue(topic.contains("Billedet befinder sig i Kort- og Billedafdelingen, Det Kongelige Bibliotek"));
+
+		// subject_name
+		Collection<Object> subjectName = record.getFieldValues("subject_name");
+		assertEquals(1,subjectName.size());
+		assertTrue(subjectName.contains("Egede, Poul Hansen"));
+
+		// subject_full_name
+		Collection<Object> subjectFullName = record.getFieldValues("subject_full_name");
+		assertEquals(1,subjectFullName.size());
+		assertTrue(subjectFullName.contains("Poul Hansen Egede"));
+
+		// subject_family_name
+		Collection<Object> subjectFamilyName = record.getFieldValues("subject_family_name");
+		assertEquals(1,subjectFamilyName.size());
+		assertTrue(subjectFamilyName.contains("Egede"));
+
+		// subject_given_name
+		Collection<Object> subjectGivenName = record.getFieldValues("subject_given_name");
+		assertEquals(1,subjectGivenName.size());
+		assertTrue(subjectGivenName.contains("Poul Hansen"));
+
+		// subject_date_of_birth
+		Collection<Object> subjectDateOfBirth = record.getFieldValues("subject_date_of_birth");
+		assertEquals(1,subjectDateOfBirth.size());
+		assertTrue(subjectDateOfBirth.contains("1708-0-0"));
```

Comment:
Where does this `1708-0-0` come from? The [ISO 8601 General principles](https://en.wikipedia.org/wiki/ISO_8601#General_principles) states that partial dates are stated by truncation, not by using `0` as placeholder.

Reply:
Tell this to the metadata. It is one of the broken formats I have to parse. This unittest just tests the string values read from the metadata.
The XSLT enricher will make it into nice formats.

Reply:
Grudgingly accepted.